ui/components/Wallet/WalletAnalyticsNotificationBanner.tsx

```diff
@@ -41,7 +41,8 @@ export default function WalletAnalyticsNotificationBanner(): ReactElement {
               style={{ height: "auto", margin: "8px 0" }}
               size="medium"
               type="tertiary"
-              linkTo="/settings/analytics"
+              // FIXME temporary solution to hide analytics screen
+              // linkTo="/settings/analytics"
```

Comment:
This will disable the link on the analytics notification, but to a user this will appear to be a broken link. Let's disable this notification from showing at all.